_drafts/newsletter-003.md

```diff
@@ -96,26 +102,557 @@ _Discussions:
 [sulis-lore]: https://sulisgame.com/lore
 [sulis-src]: https://github.com/Grokmoo/sulis
 
+### Veloren
+
+![TODO: local, desc](https://media.discordapp.net/attachments/449654102553788417/631209513336438794/screenshot_1570562280617.png?width=1151&height=685)
+
+[Veloren][veloren] is an open-world, open-source multiplayer voxel RPG.
+The game is in an early stage of development, but is playable.
+
+**TODO**: 0.4 was released.
+
+**TODO:** mention "Player survey"
+
+**TODO**: Here is a new video highlighting different structures in Veloren: <https://www.youtube.com/watch?v=iwP7SXdWcTg>
+
+**TODO**: Render Analysis #2 by @Vercidium
+
+rivers!
+
+**TODO**: another img with items or UI?
+
+Full weekly devlogs "This Week In Veloren...":
+[#36](https://veloren.net/devblog-36),
+[#37](https://veloren.net/devblog-37),
+[#38](https://veloren.net/devblog-38),
+[#39](https://veloren.net/devblog-39).
+
+[veloren]: https://veloren.net
+
+### PF Sandbox
+
+by [@rukai]
+
+- <https://twitter.com/thisIsRukai/status/1181208687081222154> -- textures!
```

Comment:
I would prefer to have semi-equal space for games, and having to go look these links is a bit unfortunate.
Do you think we could embed a few images/gifs and maybe some words right here in the newsletter?

Reply:
Yeah, this section is just a draft atm - see the TODO list in the PR's description.